openage/assets.py

```diff
@@ -35,13 +35,23 @@ def get_assets(args):
     if args.asset_dir:
         return Directory(args.asset_dir).root
 
+    try:
+        from .build import get_build_configuration
+    except ImportError:
+        build_src_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+        global_asset_dir = '/usr/local/share/openage'
```

Comment:
not sure if hardcoding it any way is a good idea. should always be provided by the build system, think of windows, nixos and whatever distro tries using us.